kas-poky-snapd.yml

```diff
@@ -4,7 +4,7 @@ header:
 machine: qemux86-64
 distro: poky
 target:
-  - core-image-minimal
+  - snapd
```

Comment:
no image?

Reply:
Not yet, I will switch to the demo image later on, but I want to reach the state where I can fish the deployed image and use it between stages.